.github/workflows/tag_image_push.yml

```diff
@@ -17,7 +17,6 @@ on:
       RELEASE_TYPE:
         description: 'What should be built'
         required: true
-        default: 'EE'
```

Comment:
It's there for a reason, without it we would have `ALL` as the default value in the UI. Setting this value helps us avoiding OSS releases by accident

![image](https://github.com/user-attachments/assets/d3fde041-bcab-4889-b114-42fc03598410)

vs 

![image](https://github.com/user-attachments/assets/d46cc5a2-555a-4e87-9b05-c3ae7292a191)

Reply:
What sets the order in UI?

Reply:
> It's there for a reason, without it we would have `ALL` as the default value in the UI. Setting this value helps us avoiding OSS releases by accident

That makes sense, thanks.
If we were going to continue with ActionLint I'd document this nuance properly, but we aren't, so instead I'll just leave it alone.

4a78d0747be33ef05fbea54fe8dedeffd9113336

Reply:
> What sets the order in UI?

Order of items:
```yaml
        options:
          - ALL
          - OSS
          - EE
          - README
```

We could move `EE` to the top, but IMHO we would make the workflow fragile since adding a new value as a first option would break it. By making `EE` explicit we show our intention: this very option should be the default one. 

Also I've noticed that in https://jenkins.hazelcast.com/view/all/job/Hazelcast-packaging-master-snapshot-trigger/configure we don't specify release type which suggests that the default value is used there

Reply:
Makes sense. good catch
Also remembered, for Release pipeline Jenkins build, EE is always checked